File: Competitive Programming 3/Section 1.3.3/10300 - Ecological Premium/C++/main.cpp
#include<iostream>

using namespace std;

int n,farmers,sz,animals,friendliness;

int main(){
    ios_base::sync_with_stdio(0);cin.tie(0);
    cin >> n;
    while(n--){
        int64_t ans = 0 ;
        cin >> farmers ; 
        while(farmers--){
            cin >> sz >> animals >> friendliness;
            ans += (int64_t) sz * friendliness;
        }
        cout << ans << "\n";
    }
    return 0;
}

File: Competitive Programming 3/Section 1.3.3/10300 - Ecological Premium/Python/main.py
def main():
    n = int(input())
    for _ in range(0,n):
        ans = 0
        farmers = int(input())
        for _ in range(0,farmers):
            sz,animals,friendliness = map(int,input().split())
            ans += sz * friendliness
        print(ans)
main()